### PR TITLE
Module system revisions

### PR DESCRIPTION
**Pull request by aturon**
Some clarifications and consistency edits.

### CODE REVIEW

File: src/2018/transitioning/modules/macros.md [hunk 1]
```diff
@@ -1,9 +1,7 @@
 # Macro changes
 
-Now that `extern crate` is gone, there's a problem: to import macros, you
-need to put `#[macro_use]` on the `extern crate` declaration.
-
-To fix this issue, you can use `use` to import macros from extern crates.
+In Rust 2018, you can use `use` to import specific macros from extern crates,
```
Comment (Centril) [inline src/2018/transitioning/modules/macros.md]: The word "use" is repeated twice here, which feels a bit sub-optimal. Not sure what to replace it with tho.

Comment (Centril) [reply src/2018/transitioning/modules/macros.md]: Replace `extern crates` with `external crates` :)

Comment (otavio) [reply src/2018/transitioning/modules/macros.md]: What about:

In Rust 2018, the `use` keyword allows the importing of specific macros from external crates,

Comment (Centril) [reply src/2018/transitioning/modules/macros.md]: If we reorder:
> In Rust 2018, specific macros from external crates can be imported with `use`.

File: src/2018/transitioning/modules/macros.md [hunk 2]
```diff
@@ -43,4 +43,4 @@ This moves `macro_rules` macros to be a bit closer to other kinds of items.
 ## More details
 
 This only works for `extern crate`; if you write a macro in the same crate,
-nothing changes.
\ No newline at end of file
+nothing changes from Rust 2015.
```
Comment (Mark-Simulacrum) [inline src/2018/transitioning/modules/macros.md]: I thought use_extern_macros permits using macros from within a crate as well. I seem to recall recent discussions toward this being a change we're making, but I'm not completely up to date...

**Comment by aturon** [comment]
Review addressed.

**Pull request merged**